cypress/integration/user/generate.js

```diff
@@ -1,30 +1,45 @@
 describe('Generate users', () => {
   it('Should not have permissions when not logged in', () => {
-    cy.visit('http://localhost:3000/user/generate')
-    cy.get('input[id="mail"]').click().type('test@wellcode.com')
-    cy.get('input[id="name"]').click().type('test name')
-    cy.get('input[id="password"]').click().type('test password')
+    cy.request({ url: '/api/user/generate', failOnStatusCode: false })
```

Comment:
We should do a post request here with a similar payload to what the actual app is sending so we can simulate the real behavior perfectly
Also, let's test first the interface and move this line afterwards